docs/src/markdownDocs/workflows/user-auth.mdx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+# React User Authentication Workflow
+
+The React User Authentication Workflow package, `@brightlayer-ui/react-auth-workflow`, provides a consistent and comprehensive UI implementation of authentication-related capabilities for use in Eaton web applications built with React.
+
+The package is intended to provide a standard, out-of-the-box experience for capabilities such as:
+
+-   Login
+-   Forgot / Reset Password
+-   Change Password
+-   Contact Support
+-   Self Registration
+-   Invitation-based Registration
+
+This package is flexible, allowing you to use the Login and Registration flows independently or in combination (or simply use individual screen components), while also providing many opportunities to customize the flows if needed for your particular application.
+
+[Read more about this package](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow) on GitHub.
```

Comment:
Just a question, will this link be accessed out of etn-ccis?

Reply:
Yes, you can visit this link even in incognito mode.